.bazelrc

```diff
@@ -89,6 +89,7 @@ build:generic_clang --copt=-Wno-unused-function
 build:generic_clang --copt=-Wno-unused-local-typedef
 build:generic_clang --copt=-Wno-unused-private-field
 build:generic_clang --copt=-Wno-user-defined-warnings
+build:generic_clang --copt=-Wno-macro-redefined # IREE and TensorFlow define some of the same macros
```

Comment:
I think we should open and link an issue to track this. Would be nice to fix if we can.

Reply:
What sort of resolution would you like to see for it? I'm not really concerned with the current overlap.

Reply:
For having this turned off in general. I'm not super concerned with the current overlap, but I think we should have this error, if for no other reason than it's turned on internally, but also because redefining macros generally indicates something has gone wrong with layering.

Reply:
¯\\\_(ツ)\_/¯
Filed https://github.com/google/iree/issues/2556 and linked to it here